Modules/Sources/JetpackStats/Cards/ChartCard.swift

```diff
@@ -5,8 +5,6 @@ struct ChartCard: View {
     @ObservedObject private var viewModel: ChartCardViewModel
 
     private var onDateRangeSelected: ((StatsDateRange) -> Void)?
-    private var backButtonTitle: String?
```

Comment:
This button is now added from the outside of the card, and looks more like a button.

<img width="440"  alt="Screenshot 2025-09-29 at 7 21 48 AM" src="https://github.com/user-attachments/assets/4605ca17-12d7-4157-b6c0-0788009b8baf" />